testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -721,7 +721,7 @@ def pillar_get(key, minion)
 
 And(/^I cleanup minion "([^"]*)"$/) do |minion|
   node = get_target(minion)
-  if minion == 'sle_minion'
+  if %w[sle_minion sle_ssh_tunnel_minion].include?(minion)
```

Comment:
```suggestion
  if minion.include? 'minion'
```

Reply:
I think that's incorrect, this string comparaison will also catche ceos_minion etc.

Reply:
I would make it more generic like that, so next time we add a new host name we don't need to touch this line

Reply:
Well, and what's the problem to also match "ceos_minion"?
We are already using this construction in other steps, like in the "I apply highstate on"

Reply:
I see, I just not read the next elseif... well, I would reconsider the other block, are we sure we can make a unique block?

Reply:
No, that would include the second branch of the `if`.  We use `systemd` for Ubuntu/Centos minions.
See the full lines:
```
  if %w[sle_minion sle_ssh_tunnel_minion].include?(minion)
    node.run('rcsalt-minion stop')
    node.run('rm -Rf /var/cache/salt/minion')
  elsif %w[ceos_minion ceos_ssh_minion ubuntu_minion ubuntu_ssh_minion].include?(minion)
    node.run('systemctl stop salt-minion')
    node.run('rm -Rf /var/cache/salt/minion')
  end

```